test/src/test/resources/fixtures/renames/types/renamed-type-is-shared-but-not-renamed-in-one-service.yml

```diff
@@ -1,5 +1,6 @@
 name: renamed type is shared but not renamed in one service
-enabled: true
+# Not supported
```

Comment:
what's the plan with these? are we going to delete them

Reply:
I think we'll want to support it eventually so I'm keeping the test in. This is the shared rename issue where we'll want the repeatable

```
```

syntax.